src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_slr_enabled_and_transaction_mode_none.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_slr_enabled_and_transaction_mode_none : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_throw_exception_on_endpoint_startup()
```

Comment:
@timbussmann I didn't want to remove it without folks responsible for v6 scope at least acknowledging the change.